src/components/BlockingViews/FullPageNotFoundView.tsx

```diff
@@ -7,54 +6,54 @@ import useLocalize from '@hooks/useLocalize';
 import Navigation from '@libs/Navigation/Navigation';
 import useThemeStyles from '@styles/useThemeStyles';
 import variables from '@styles/variables';
+import {TranslationPaths} from '@src/languages/types';
 import ROUTES from '@src/ROUTES';
 import BlockingView from './BlockingView';
 
-const propTypes = {
+type FullPageNotFoundViewProps = {
     /** Child elements */
-    children: PropTypes.node,
+    children?: React.ReactNode;
 
     /** If true, child components are replaced with a blocking "not found" view */
-    shouldShow: PropTypes.bool,
+    shouldShow?: boolean;
 
     /** The key in the translations file to use for the title */
-    titleKey: PropTypes.string,
+    titleKey?: TranslationPaths;
 
     /** The key in the translations file to use for the subtitle */
-    subtitleKey: PropTypes.string,
+    subtitleKey?: TranslationPaths;
 
     /** Whether we should show a link to navigate elsewhere */
-    shouldShowLink: PropTypes.bool,
+    shouldShowLink?: boolean;
 
     /** Whether we should show the back button on the header */
-    shouldShowBackButton: PropTypes.bool,
+    shouldShowBackButton?: boolean;
 
     /** The key in the translations file to use for the go back link */
-    linkKey: PropTypes.string,
+    linkKey?: TranslationPaths;
 
     /** Method to trigger when pressing the back button of the header */
-    onBackButtonPress: PropTypes.func,
+    onBackButtonPress: () => void;
 
     /** Function to call when pressing the navigation link */
-    onLinkPress: PropTypes.func,
-};
-
-const defaultProps = {
-    children: null,
-    shouldShow: false,
-    titleKey: 'notFound.notHere',
-    subtitleKey: 'notFound.pageNotFound',
-    linkKey: 'notFound.goBackHome',
-    onBackButtonPress: () => Navigation.goBack(ROUTES.HOME),
-    shouldShowLink: true,
-    shouldShowBackButton: true,
-    onLinkPress: () => Navigation.dismissModal(),
+    onLinkPress: () => void;
 };
 
 // eslint-disable-next-line rulesdir/no-negated-variables
-function FullPageNotFoundView({children, shouldShow, titleKey, subtitleKey, linkKey, onBackButtonPress, shouldShowLink, shouldShowBackButton, onLinkPress}) {
+function FullPageNotFoundView({
+    children = null,
+    shouldShow = false,
+    titleKey = 'notFound.notHere',
+    subtitleKey = 'notFound.pageNotFound',
+    linkKey = 'notFound.goBackHome',
+    onBackButtonPress = () => Navigation.goBack(ROUTES.HOME),
+    shouldShowLink = true,
+    shouldShowBackButton = true,
+    onLinkPress = () => Navigation.dismissModal,
```

Comment:
Why have you removed the function call?
```suggestion
    onLinkPress = () => Navigation.dismissModal(),
```